server/apps/publish/archive_publish.py

```diff
@@ -124,19 +124,23 @@ def update(self, id, updates, original):
                         queued_digital, takes_package = self._publish_takes_package(package_id, updates,
                                                                                     original, last_updated)
                     else:
-                        # if item is going to be sent to digital subscribers, package it as a take
-                        if self.sending_to_digital_subscribers(updates):
-                            updated = copy(original)
-                            updated.update(updates)
-                            # create a takes package
-                            package_id = TakesPackageService().package_story_as_a_take(updated, {}, None)
-                            original = get_resource_service('archive').find_one(req=None, _id=original['_id'])
-                            queued_digital, takes_package = self._publish_takes_package(package_id, updates,
-                                                                                        original, last_updated)
+                        # if text or preformatted item is going to be sent to digital subscribers, package it as a take
+                        if self.sending_to_digital_subscribers(original):
+                            # takes packages are only created for these types
+                            if (original['type'] == CONTENT_TYPE.TEXT or original['type'] == CONTENT_TYPE.PREFORMATTED):
```

Comment:
If you feel this line is a bit lengthy, you can shorten it to the following (esp. useful if there are 3 or more checks):
```py
original['type'] in (CONTENT_TYPE.TEXT, CONTENT_TYPE.PREFORMATTED)
```
No need to change it in this particular case though, just a tip.